lib/build.js

```diff
@@ -408,6 +418,20 @@ export default {
           }
           GoogleAnalytics.sendEvent('build', 'failed');
         }
+        for (const error of this.errorMatcher.currentMatch) {
+          this.linter.setMessages([{
+            type: error.type || 'Error',
+            name: targetName,
```

Comment:
I'm adding this message-specific name to linter now. It'll be in the next release.